tias/src/Components/EmployeeList/EmployeeList.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from 'react'
+import { EmployeeRow } from './EmployeeRow'
+import { GenerateButton } from './GenerateButton'
+import { AcceptButton } from './AcceptButton'
+
+interface Props {
+	
+}
```

Comment:
It's been a while since I did React. Is there a purpose to an empty `Props` interface?

Reply:
So this is future proofed.